compiler/out.js

```diff
@@ -0,0 +1,290 @@
+const r = require('alan-js-runtime')
```

Comment:
I don't think you meant to add this to the PR...

Reply:
yes, removed!